libraries/AP_DroneCAN/AP_DroneCAN.h

```diff
@@ -145,6 +145,9 @@ class AP_DroneCAN : public AP_CANDriver, public AP_ESC_Telem_Backend {
     void SRV_send_esc();
     void SRV_send_himark();
 
+    //scale servo output appropriately before sending
+    int scale_esc_output(uint8_t idx);
```

Comment:
Makes sense. Since the DroneCAN RawCommand is a [14 bit integer](https://dronecan.github.io/Specification/7._List_of_standard_data_types/#rawcommand), an int16_t should be sufficient.